include/openenclave/internal/datetime.h

```diff
@@ -58,6 +58,16 @@ oe_result_t oe_datetime_now(oe_datetime_t* value);
  */
 void oe_datetime_log(const char* msg, const oe_datetime_t* date);
 
+/**
+ * Convert oe datetime to time_t.
+ */
+oe_result_t oe_datetime_to_time_t(const oe_datetime_t* datetime, time_t* value);
+
+/**
+ * Convert time_t to oe datetime.
```

Comment:
```suggestion
 * Convert time_t to oe datetime.
 * NOTE: This function is insecure, since it relies on untrusted time information from the host.
```

Reply:
Same comments as above